locale/en_US/LC_MESSAGES/clr-installer.po

```diff
@@ -812,3 +812,23 @@ msgstr "Network Required"
 
 msgid "Installation Steps Complete"
 msgstr "Installation Steps Complete"
+
+#, c-format
+msgid "Password is too similar to the user attribute: %s"
```

Comment:
Is this translation still used?  I can't find a reference to it.

Reply:
Thanks for finding this..removed it